webrtc.html

```diff
@@ -509,33 +509,47 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>If <code>operations</code> is empty, execute the
+              function and let <var>p</var> be the promise returned by
+              the function.  If <code>operations</code> is non-empty,
+              let <var>p</var> be a promise equivalent to the promise
+              that would be returned by calling the function (it will
+              be settled at the same time and with the same result).
+            </p>
+          </li>
+
+          <li>
+            <p>Add <code>p</code> to <var>operations</var>.</p>
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>When a promise in <code>operations</code> is settled,
```

Comment:
Again, I think the "when" needs to be replaced with steps that the dictate specific steps around promises that must be taken. I've found that people care about such timing behaviors being nailed down.

E.g. that's why a lot of input validation was recently moved to the synchronous parts of methods in media capture (people can inspect a promise returned from a function in the browser's JS debugger and tell that input validation failed from the fact that the promise already shows as rejected).